docs/index.html

```diff
@@ -1 +1,35 @@
-Test
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog Post</title>
+  </head>
+  <body>
+    <img src="https://images.unsplash.com/photo-1546074177-ffdda98d214f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80" alt="Describe what's in the photo">
```

Comment:
«Describe what's in the photo» is not the best description ;)

Reply:
Fixed ;)